content/content-strategy/define-user-content-needs/cards.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: cards/cards
+cards:
+  - headline: Understand users and their needs
+    text: Knowing your users and their needs is the starting point for any product or service.
+    link: https://guides.service.gov.au/user-research/identifying-users-needs/
```

Comment:
Needs to be a relative path, i.e. *not* with 'https://guides.service.gov.au' in front.

All Guides topics referenced in cards should be relative links only.

Reply:
Steve fixed